tools/wpt/wpt.py

```diff
@@ -131,6 +132,15 @@ def setup_virtualenv(path, skip_venv_setup, props):
 
 def main(prog=None, argv=None):
     logging.basicConfig(level=logging.INFO)
+    # Ensure we use the spawn start method for all multiprocessing
+    try:
+        multiprocessing.set_start_method('spawn')
+    except RuntimeError as e:
+        # This can happen if we call back into wpt having already set the context
```

Comment:
This should be possible to avoid if we put it in the root `wpt` right?

Reply:
I *think* so. But the reason I noticed this is that we have unit tests that call `wpt.main` instead of going via the command line. So I think we'd run the risk of those tests running under an unsupported configuration if we so this in a pre-main location.

I tried a test and I can't see that calling this in `main` vs `if __name__ == "__main__"` makes any difference to the behaviour; in each case the subprocess gets the same default spawn method. It seems like the important thing is really that it's called at the main entrypoint to the code; as long as we're confident everything goes through `main` it's all fine.

Reply:
Yeah, probably the Python documentation says that as a slightly inaccurate way of saying "please make sure to call it always and early".